<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>598</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RSA</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../GitHub/tbu-affine-cipher/assets/lock_icon.svg</normaloff>../../../GitHub/tbu-affine-cipher/assets/lock_icon.svg</iconset>
  </property>
  <property name="locale">
   <locale language="Czech" country="CzechRepublic"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutTop">
       <item>
        <widget class="QGroupBox" name="groupBoxInput">
         <property name="title">
          <string>Vstup</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>29</y>
            <width>321</width>
            <height>291</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayoutInput">
           <item row="4" column="1">
            <widget class="QPlainTextEdit" name="textEditInput"/>
           </item>
           <item row="5" column="1">
            <layout class="QHBoxLayout" name="horizontalLayoutDoStuff">
             <item>
              <widget class="QPushButton" name="buttonDoStuff">
               <property name="text">
                <string>Proveď</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="spacerDoStuff">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="labelInput">
             <property name="text">
              <string>Zadejte text k zašifrování:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxSettings">
         <property name="title">
          <string>Nastavení</string>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>151</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="labelOption">
             <property name="text">
              <string>Režim:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QRadioButton" name="radioButtonEncrypt">
               <property name="text">
                <string>Šifrování</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoRepeat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonDecrypt">
               <property name="text">
                <string>Rozšifrování</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>321</width>
            <height>171</height>
           </rect>
          </property>
          <property name="title">
           <string>Klíče</string>
          </property>
          <widget class="QWidget" name="gridLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>301</width>
             <height>141</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditPubN"/>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="lineEditPubE"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEditPrivN"/>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="lineEditPrivD"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelPriv">
              <property name="text">
               <string>Soukromý:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="labelPub">
              <property name="text">
               <string>Veřejný:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QPushButton" name="buttonOpenKeyGen">
              <property name="text">
               <string>Generovat</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>689</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Výstup</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>661</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelOutput">
        <property name="text">
         <string>Zašifrovaný text:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="textEditOutput"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
